### ISSUE TITLE
NCBIGene species

### ISSUE THREAD

**Issue by tgbugs** [SciCrunch/NIF-Ontology#66]
We need to select a default reference species for all NCBIGene identifiers since there aren't ortholog group identifiers. This does result in the question as to what identifiers we should use for expression phenotypes (this becomes very important when communicating about knockout/rescue experiments where one putative ortholog is replaced with another, eg mouse cells expressing a human gene). See for example CCK/Cck.

http://www.ncbi.nlm.nih.gov/gene/885

http://www.ncbi.nlm.nih.gov/gene/12424

**Comment by tgbugs** [comment]
An addition question about NCBIGene is what to do when they remove previously accepted synonyms.

**Comment by cmungall** [comment]
Are you talking about having a species-generic class for genes, 
analogous to what PRO does?

I would recommend using NCBIGene for species-level, and Panther for 
ancestors. I don't know if PRO have plans for species-generic genes in 
generic protein definitions

**Comment by tgbugs** [comment]
We sometimes need species generic identifiers in order to talk about expression for neurons, if an experiment detects protein expression then PR has us covered, but if it detects DNA or RNA then it would be nice to use an id that refers to a gene. If Panther has identifiers for ancestral genes that correspond to ortholog groups that would be perfect.

**Comment by tgbugs** [comment]
For example this is perfect for cck http://www.pantherdb.org/panther/family.do?clsAccession=PTHR10786. The only issue is that the mappings to NCBIGene are a bit opaque, but it shouldn't be too hard to deal with that.

**Comment by tgbugs** [comment]
NCBIGene:16847 is pulling in diabetes as a synonym, rather problematic. Should tweak slimgen.py to filter those out as well as shifting the short synonyms to acronyms. There may be more issues like this.

**Comment by cmungall** [comment]
Let's work together on that! cc @kltm @jjnguyenx

**Comment by tgbugs** [comment]
Definitely. I'm not going to be able to dig into this until October (thesis proposal coming up) but my plan would be to add a section to [slimgen.py](https://github.com/tgbugs/pyontutils/blob/master/slimgen.py) that takes the NCBIGene identifiers as inputs and finds the set of panther ids that cover that NCBIGene ids (currently listed [here](https://github.com/tgbugs/pyontutils/blob/master/gene-subset-ids.txt)).

Information about how to find the right ID in the NCBI system so that it can also be located in panther is what I am missing at the moment. I *think* that panther uses the primary identifier from the NCBI data record so if we get that we might be able to do the mapping that way.
